packages/sdk/src/interaction/interaction.js

```diff
@@ -138,6 +138,16 @@ const makeIx = wat => ix => {
   return Ok(ix)
 }
 
+const prepAccountKeyId = acct => {
+  if (acct.keyId == null || acct.keyId == undefined) return acct
```

Comment:
```suggestion
  if (acct.keyId == null) return acct
```

Given you are not using strictly equal you only need to check against null.